src/plugins/wizard/public/application/components/data_tab/config_panel.scss

```diff
@@ -0,0 +1,9 @@
+.wizConfigPanel {
+  background: #f0f1f3;
```

Comment:
Is this color shared anywhere else?

Reply:
Not this version. I've since spoken to @btzeng and he had updated the colors to match the ones from eui. I'll be making that and some other UI fixes in a follow up PR